tests/TestRunner/app/Marshalling/ObjCTypesTests.js

```diff
@@ -178,4 +178,27 @@ describe(module.id, function () {
     it("NSNull", function () {
         expect(NSNull.null() instanceof NSObject).toBe(true);
     });
+         
+    it("AllocatedPlaceholderHasOnlyOneSymbol", function () {
```

Comment:
Why do we need this test? It only begs to be broken if we add another symbol in the future.